apis/swagger.yml

```diff
@@ -567,6 +581,33 @@ definitions:
       message:
         type: string
 
+  DragonflyVersion:
+    type: "object"
+    description: |
+      Version and build information from supernode.
+    properties:
+      version:
+        type: "string"
+        description: "Version of supernode"
+      revision:
+        type: "string"
+        description: "Git commit when building supernode"
+      branch:
+        type: "string"
+        description: "Git branch when building supernode"
+      buildDate:
+        type: "string"
+        description: "Build Date of supernode"
+      goVersion:
+        type: "string"
+        description: "Golang runtime version"
+      goOS:
```

Comment:
Why to add a prefix of `go` in `goOS`? It is some kind of weird. I think `OSType` is fine, since OS type is not related to go runtime.